### ISSUE TITLE
messages sent by other Facebook clients are not showing up in Matrix conversation

### ISSUE THREAD

**Issue by woojoo666** [mautrix/facebook#205]
Sending messages via Matrix works fine, but when I send a message via Facebook, it doesn't show up in the corresponding Matrix chat history. The logs show

```
Error in incoming MQTT message handler
Traceback (most recent call last):
  File "/opt/mautrix-facebook/maufbapi/mqtt/conn.py", line 310, in _on_message_handler
    topic = RealtimeTopic.decode(message.topic)
  File "/opt/mautrix-facebook/maufbapi/mqtt/subscription.py", line 44, in decode
    topic = _reverse_topic_map[val]
KeyError: '273'
```

I am using version v0.3.2 (docker build). I noticed in [topics.json](https://github.com/mautrix/facebook/blob/v0.3.2/maufbapi/mqtt/topics.json) that there is no mapping for `273`. Not sure where these mappings are coming from but if they are based on Facebook's API maybe they are outdated?

Note that I'm currently _not_ using double-puppetting since I haven't set it up yet, but would that fix this error?

**Comment by tulir** [comment]
The error just means you're an old version of the bridge, it doesn't break anything. Double puppeting is required for your own messages to show up.

**Comment by woojoo666** [comment]
I tried re-deploying using the latest docker image (`dock.mau.dev/mautrix/facebook:latest`) and while the error message from above didn't seem to appear anymore, I still wasn't seeing any messages that I sent from other facebook clients. It was only after I set up double-puppetting that it started working. So I guess the "Matrix ghost of your remote account" mentioned [in the mautrix docs](https://docs.mau.fi/bridges/general/double-puppeting.html) isn't supported on mautrix-facebook?
